src/main/java/com/openlattice/auth0/Auth0TokenProvider.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2017. OpenLattice, Inc
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * You can contact the owner of the copyright at support@openlattice.com
+ *
+ */
+
+package com.openlattice.auth0;
+
+import com.auth0.client.auth.AuthAPI;
+import com.auth0.exception.Auth0Exception;
+import com.auth0.json.auth.TokenHolder;
+import com.auth0.net.AuthRequest;
+import com.openlattice.authentication.Auth0Configuration;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.scheduling.annotation.Scheduled;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+public class Auth0TokenProvider {
+
+    private static final Logger logger = LoggerFactory.getLogger( Auth0TokenProvider.class );
+
+    static final        int    CHECK_EXP_INTERVAL_MILLIS   = 60 * 60 * 1000; // 1 hour
+    public static final String AUTH0_MANAGEMENT_API_V2_URL = "https://openlattice.auth0.com/api/v2/";
```

Comment:
For rhizome make this an argument that is passed in and a constant where ever it is being invoked.

Reply:
Done. `managementApiUrl` will now be a required field in the yaml config.